ui/components/multichain/address-list-item/index.scss

```diff
@@ -0,0 +1,11 @@
+.address-list-item {
+  /* Set explicitly to match the font-size of AccountListItem */
+  .mm-text {
+    font-size: 14px !important;
```

Comment:
Hmm..why do we need to fix it to 14px? Can we use bodySm?

Reply:
It corresponds to this:

<img width="927" alt="SCR-20231108-krwr" src="https://github.com/MetaMask/metamask-extension/assets/46655/86e9baea-dccb-470b-ae0e-a18a3c600663">

Otherwise in full screen, it looks like this (larger font):

<img width="403" alt="SCR-20231108-ksjf" src="https://github.com/MetaMask/metamask-extension/assets/46655/5b71725b-bfbb-41bf-896c-4264bdd8db61">